rust/src/storage/file/file_storage_manager.rs

```diff
@@ -122,20 +136,24 @@ where
 #[cfg(test)]
 mod tests {
     use std::{
-        fs::{self, File, Permissions},
+        fs::{self, Permissions},
         os::unix::fs::PermissionsExt,
     };
 
-    use zerocopy::IntoBytes;
-
     use super::*;
-    use crate::{storage::file::SeekFile, types::NodeId};
-
-    type FileStorageManager = super::FileStorageManager<SeekFile>;
-    type NodeFileStorage = super::NodeFileStorage<InnerNode, SeekFile>;
+    use crate::{
+        storage::file::{NodeFileStorage, SeekFile},
+        types::NodeId,
+    };
 
     #[test]
-    fn open_creates_files_if_they_do_not_exist() {
+    fn open_creates_directory_and_opens_committed_checkpoint_and_calls_open_on_all_storages() {
```

Comment:
```suggestion
    fn open_creates_directory_and_calls_open_on_all_storages() {
```
There are still no checkpoints